### ISSUE TITLE
Beginners Contibuting Guidelines.

### ISSUE THREAD

**Issue by GrayFlash** [CrewMates-Open-Source/EduHelp#4]
While we are setting up guidelines for contributors, we will wish a participant to set-up the dev environment locally. It should be one of the first things you do with any repository you want to contribute in.

It involves the steps from :
* forking and cloning repository locally
* exploring branches, here main has our sample code and gssoc21 is a clean branch which we will use throughout the development period
* creating a venv environment named env (please use same name as we wish to avoid multiple environments from various pull requests).
* running the code locally after compilation.

So we wish all participants to perform this tasks before starting with issues, don't worry if issues are being assigned to someone else, we have enough development to do. 

One of the participant needs to create a .md file named starter.md so yours peers don't find it difficult to get started. Include all the steps from fork, to clone and checkout branches. commands that you ran on your terminals in same manner.

**Comment by Anurag-bhatia** [comment]
I would like to contribute on this issue . Kindly assign me this issue under the GSSOC 21.

**Comment by GrayFlash** [comment]
@Anurag-bhatia start setting up the environment, I have asked someone in #3  for doing the same, though if you are done earlier than him and he does reply by then, I will assign it to you.

Setting up the environment is the first step though, so get started.

**Comment by Abhi-1313** [comment]
@GrayFlash assign me this

**Comment by Anurag-bhatia** [comment]
@GrayFlash I setup it up in my local system , could you assign me now , so i can start working.

**Comment by GrayFlash** [comment]
I will assign this issue to @Abhi-1313 , cause He was looking for issues befor @Anurag-bhatia .



> @GrayFlash I setup it up in my local system , could you assign me now , so i can start working.

That's great, we will soon make issues for implementing of utils file written in python, so you can have a look at out code base in main branch it is a bit messy and avoid wondering about tkinter for this moment. We will need people to implement seperate utils that address the recording of audio, capturing distinct screenshots, saving them to files and later for processing this audio and content recording. We will then move ahead with speech to text conversion, identifying different people on basics of vocal signals and segmenting those and creating a summary of meeting.

This is our overall objective of project in terms of funtionalities and we will discuss it and make it formal. We will also create some basic GUI for operating the entire utils later on.

I am just adding this content here so if any participant want to know obout project can refer this.

**Comment by GrayFlash** [comment]
@Anurag-bhatia if you have any suggestion after going through code base and if you want to contribute and send a pr you can create an issue.

**Comment by GrayFlash** [comment]
@Anurag-bhatia are you still looking to work on this issue?

**Comment by Anurag-bhatia** [comment]
@GrayFlash Yes , I will do this if you assigned me.

**Comment by GrayFlash** [comment]
Ok I will create a new issue with things that are missing and you complete those, @Anurag-bhatia

**Comment by GrayFlash** [comment]
@Anurag-bhatia visit issue number #13 and I'll assign that to you
